notebooks/book1/02/softmax_plot.ipynb

```diff
@@ -0,0 +1,185 @@
+{
```

Comment:
<p>Add title of notebook, observe other notebooks</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/59/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
okay. understood.